decentraland/unity-explorer
# fix: edge case for missing materials in wearables

## What does this PR change?

We found this pants that do not have a material in one of its SMR. Looks like a creator error

To avoid this issue from occuring, we add a null-check and create a new material if needed

## How to test the changes?

1. Launch the explorer
2. Instantiate this wearable (urn:decentraland:matic:collections-v2:0xabee95fac242e222b100b319708cbdfec2c5c86a:0) using the debug menu
3. Check that the pants are created!

![image](https://github.com/decentraland/unity-explorer/assets/1999557/042bb400-a2d7-4499-8305-e7b74272da81)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
